specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/integrations.json

```diff
@@ -0,0 +1,737 @@
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Create or update integration over subscription scope": {
+            "$ref": "./examples/Integrations/PutIntegration_example.json"
+          },
+          "Create or update integration over security connector scope": {
+            "$ref": "./examples/Integrations/PutSecurityConnectorIntegration_example.json"
+          },
+          "Create or update integration over management group scope": {
+            "$ref": "./examples/Integrations/PutManagementGroupIntegration_example.json"
+          }
+        },
+        "tags": [
+          "Integrations"
+        ],
+        "description": "Creates or updates an integration over a given scope",
+        "operationId": "Integrations_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/IntegrationId"
+          },
+          {
+            "$ref": "#/parameters/IntegrationBody"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Updated",
+            "schema": {
+              "$ref": "#/definitions/Integration"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/Integration"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Delete an integration over subscription scope": {
+            "$ref": "./examples/Integrations/DeleteIntegration_example.json"
+          },
+          "Delete an integration over security connector scope": {
+            "$ref": "./examples/Integrations/DeleteSecurityConnectorIntegration_example.json"
+          },
+          "Delete an integration over management group scope": {
+            "$ref": "./examples/Integrations/DeleteManagementGroupIntegration_example.json"
+          }
+        },
+        "tags": [
+          "Integrations"
+        ],
+        "description": "Delete an integration over a given scope",
+        "operationId": "Integrations_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/IntegrationId"
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
How come the delete operation is done async but the PUT request isn't (defined as sync)? this doesn't make sense to me